api/py/ai/chronon/scheduler/adapters/airflow_adapter.py

```diff
@@ -19,7 +16,7 @@ def __init__(self, dag_id, start_date, schedule_interval="@once", airflow_cluste
         self.airflow_cluster = airflow_cluster
 
     def setup(self):
-        airflow_client.init(self.airflow_cluster)
+        """Initialize a connection to Airflow"""
```

Comment:
Did you mean to delete the airflow_client code here? Do you want to log the string?

Reply:
We will move `airflow_client` import internal. Then this `airflow_adapter` can be used for both adhoc runs (triggered by airflow_client) and production runs (scheduled by Airflow)